bot/exts/evergreen/gurkan.py

```diff
@@ -0,0 +1,29 @@
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+
+class Gurkan(commands.Cog):
+    """Commands related to Gurkan and the Gurkult."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="gurkancount", aliases=("gc", "gurkcount"))
+    async def gurkancount(self, ctx: commands.Context) -> None:
```

Comment:
This function name could be clearer if it was using `snake_case` i.e `gurkan_count`

Reply:
I'm not sure about this one, since it's a command. If it was a usual method, it would be using snake_case, but this is a command.

Reply:
What's wrong with having the name as `snake_case`?

Reply:
We definitely could. I didn't think that was the convention for commands, though. Let's do it.

Reply:
The command name is `gurkancount`, as you set it in the decorator, but the method name should be `snake_case` instead.